tests/functional/all_daemons/hosts

```diff
@@ -32,7 +32,7 @@ client1
 rbd-mirror0
 
 [iscsigws]
-iscsi-gw0 ceph_repository="dev"
```

Comment:
You still need to set the variable here otherwise it will fallback to `ceph_repository: dev` via the group_vars